### PR TITLE
Update UX for post long press menu

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Updates the UX for the post context menu on long press and takes out the options for copying URL, code and mention to a bottom sheet

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-12213

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information
This PR was tested on: iOS and Android

#### Screenshots
Bottom Sheet
simulator screen shot - iphone xs - 2018-11-06 at 19 32 49

Menu on a message posted by me
simulator screen shot - iphone xs - 2018-11-06 at 19 33 25

Menu on a message posted by others
simulator screen shot - iphone xs - 2018-11-06 at 19 35 51

### CODE REVIEW

**Comment by enahum** [comment]
@esethna I'll take a look at 1 and 2 but 3 you should discuss in UX, I followed the spec and I don't want to change it unless the spec is updated

**Comment by esethna** [comment]
^ Discussed with AM, we're fine moving it to the bottom since it's a destructive action and consistency with such actions is a good thing. I updated the spec

**Comment by enahum** [comment]
@esethna adjustments made

**Comment by esethna** [comment]
@enahum ^ the above test build seems to have all the same issues. Wondering if it pulled the updates correctly?

**Comment by enahum** [comment]
@esethna I’m rebuilding this but by using the link in the quality assurance channel is working as expected for me, wondering if the issue was the link in GH? Anyway try that one

**Comment by enahum** [comment]
@esethna were you able to test this again?

File: app/components/file_attachment_list/file_attachment_list.js [hunk 1]
```diff
@@ -100,7 +99,8 @@ export default class FileAttachmentList extends Component {
                 }
 
                 if (cache) {
-                    uri = cache.path;
+                    const prefix = Platform.OS === 'android' ? 'file://' : '';
```
Comment (hmhealey) [inline app/components/file_attachment_list/file_attachment_list.js]: Is this something that needs to be fixed on the release branch? I forget where those changes were done?

Comment (enahum) [reply app/components/file_attachment_list/file_attachment_list.js]: I think it already was, wasn’t it?

Comment (saturninoabril) [reply app/components/file_attachment_list/file_attachment_list.js]: Yup, it's there on the release branch.

**Comment by enahum** [comment]
I’ll resolve the conflicts and then merge

**Pull request merged**